packages/common/src/ArcGISContext.ts

```diff
@@ -169,8 +171,20 @@ export interface IArcGISContext {
    * Derived from properties.alphaOrgs
    */
   isAlphaOrg: boolean;
+
+  environment: HubEnvironment;
+
+  features: IHubFeatures;
```

Comment:
ArcGISContext now has `environment` and `features` (maybe this should be called `featureOverrides`).